umibukela/views.py

```diff
@@ -46,11 +45,40 @@ def site(request, site_slug):
     })
 
 
+def site_result_hardcoded(request, site_slug, result_id):
+    result_set = CycleResultSet.objects.get(
+        id=result_id,
+        site__slug__exact=site_slug
+    )
+    return render(request, 'site_result_hardcoded.html', {
+        'active_tab': 'sites',
+        'result_set': result_set,
+    })
+
+
 def site_result(request, site_slug, result_id):
-    result_set = CycleResultSet.objects.get(id=result_id, site__slug__exact=site_slug)
+    result_set = CycleResultSet.objects.get(
+        id=result_id,
+        site__slug__exact=site_slug
+    )
+    site_submissions = Submission.objects.filter(cycle_result_set=result_set)
+    site_responses = [submission.answers for submission in site_submissions]
```

Comment:
It's pretty common in python to use short variable names in list comprehensions, because it keeps them short. eg `site_responses = [s.anwsers for s in site_submissions]`